use-case-guides/workday-request-time-off/README.md

```diff
@@ -208,150 +183,330 @@ curl --location 'https://{{domain}}.workday.com/ccx/api/absenceManagement/v1/{{i
+    try:
+        access_token = await get_access_token()
+        raw_data = await fetch_worker_details(email, access_token)
+        processed_data = flatten_worker_details_json(raw_data)
+        return FlattenedWorkerResponse(total=len(processed_data), data=processed_data)
+    except httpx.HTTPError as e:
+        raise HTTPException(
+            status_code=500, detail="Failed to fetch worker details"
+        ) from e
+
+
+@app.post("/workers/{email}/time-off")
+async def create_time_off_request_endpoint(
+    email: str, request: TimeOffRequest
+):
+    try:
+        access_token = await get_access_token()
+        worker_data = await fetch_worker_details(email, access_token)
+        if not worker_data:
+            raise HTTPException(status_code=404, detail="Worker not found")
+
+        worker = worker_data[0]
+        days = create_time_off_days(
+            request, worker["position"]["id"], worker["position"]["descriptor"]
+        )
+        response = await submit_time_off_request(
+            worker["workdayID"], days, access_token
+        )
+        return JSONResponse(content=response, status_code=201)
+    except httpx.HTTPError as e:
+        raise HTTPException(
+            status_code=500, detail="Failed to create time off request"
+        ) from e
+```
 
-**🗣️ Natural Language Slots**
+## Step 4: Build in Creator Studio
 
-1. Start Date: when time off starts
-2. End Date *(optional)*: when time off ends. By default end date is the same as the start date.
-3. Comments *(optional)*: Any relevant context / justification for taking time off
-4. Start Time *(optional)*: if taking partial days off, starting time for time off can be specified.
-5. End Time *(optional)*: if taking partial days off, ending time for time off can be specified.
+We will be building this plugin as a Query Resolver (fka Query-Triggered Path) along with an iPaaS to chain APIs 1, 2, and 3 together in order to submit a time off request for a user.
 
-🤲 **Inferred Slots**
+### Plugin Architecture Diagram
 
-1. Current Date: used in the "as of date" in the request PTO action. Could be static.
-2. Email Address: inferred from user roster profile of the employee calling this plugin
+![Untitled](./Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/Untitled.png)
 
-**🗺️ Resolvers**
+**Triggers**
+
+1. Natural language input to copilot requesting to take time off
 
-1. Time Off Plan: maps an employee’s time off plans to the corresponding time off types (which is required by the request time off API)
+**Natural Language Slots**
 
-**🏃‍♂️ Actions**
+1. Start Date: when time off starts. Configured as a Date Slot.
+2. End Date *(optional)*: when time off ends. By default end date is the same as the start date if the user does not specify an end date. Configured as a Date Slot.
+3. Comments *(optional)*: Any relevant context / justification for taking time off. Configured as a Free Text Slot.
```

Comment:
Really weird Title Case usage for Date Slot, Free Text Slot, tc.

Reply:
Visually should we be bolding the Triggers, Slots, & Actions? Up to you, but incorporate into style guidelines

Reply:
recommend bolding Triggers Slots Actions. Using Normal casing rules described before for these slot types